packages/web-react/src/components/Field/ValidationText.tsx

```diff
@@ -1,28 +1,51 @@
-import React, { ElementType } from 'react';
+import React, { ElementType, useEffect } from 'react';
 import { ValidationTextProp } from '../../types';
+import { RegisterType } from './useAriaIds';
 
 export interface ValidationTextProps extends ValidationTextProp {
   className?: string;
   elementType?: ElementType;
+  id?: string;
+  registerAria?: RegisterType;
 }
 
 const defaultProps = {
   className: undefined,
   elementType: 'div',
+  id: undefined,
+  registerAria: undefined,
 };
 
 export const ValidationText = (props: ValidationTextProps) => {
-  const { className, validationText, elementType: ElementTag = 'div' } = props;
-
-  return Array.isArray(validationText) ? (
-    <ul className={className}>
-      {validationText.map((item) => (
-        <li key={`validationText_${item}`}>{item}</li>
-      ))}
-    </ul>
-  ) : (
-    <ElementTag className={className}>{validationText}</ElementTag>
-  );
+  const { className, validationText, elementType: ElementTag = 'div', id, registerAria } = props;
+
+  useEffect(() => {
+    if (registerAria) {
+      registerAria({ add: id });
+
+      return () => {
+        registerAria({ remove: id });
+      };
+    }
+
+    return undefined;
```

Comment:
Is it necessary to return an `undefined` here? The cleanup function is undefined by default, IMHO, so I think it is not necessary here. https://react.dev/reference/react/useEffect#connecting-to-an-external-system

Reply:
if there is not return i get lint warning: 'Not all code paths return a value.'

Reply:
In this case, it is a problem with Typescript configuration. The function without return implicitly returns undefined, so it is useless to return undefined anyway.

From React useEffect documentation is clear that React can handle return of undefined form useEffect. So from my point of view returning any other thing then cleanup function is unnecessary. See https://stackoverflow.com/questions/74998189/is-it-valid-to-use-the-return-keyword-in-a-useeffect-to-stop-code-execution

So I am suggesting removing the `return undefined` and placing a disable comment for typescript here with a link to documentation of useEffect and StackOverflow with additional comment that we should set `noImplicitReturns` in `tsconfig.json` in a future, see https://bobbyhadz.com/blog/typescript-not-all-code-paths-return-a-value#:~:text=The%20error%20%22Not%20all%20code,to%20false%20in%20your%20tsconfig.

Reply:
I found out there was a bad condition, now it is without warnings and we don't need any disable comments.